build.gradle

```diff
@@ -17,19 +17,18 @@ checkstyle {
 }
 
 java {
-	sourceCompatibility = JavaVersion.VERSION_1_8
-	targetCompatibility = JavaVersion.VERSION_1_8
+	sourceCompatibility = JavaVersion.VERSION_1_9
+	targetCompatibility = JavaVersion.VERSION_1_9
```

Comment:
I'm not sure I want to require Java 9 for development. Maybe we can just remove this section?

Reply:
Done